scripts/nat/start-nat-node.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if [ $(id -u) -ne 0 ] ; then
+  >&2 echo "ERROR: Must run as root"
+  exit 1
+fi
+
+if [ -z "$HOPR_RELEASE" ]; then
```

Comment:
Please use `HOPRD_` instead of `HOPR_` since all these variables are in reference to `hoprd` configuration and not `hopr` protocol.